custom/icds_reports/templates/icds_reports/icds_app/cas-export.directive.html

```diff
@@ -4,7 +4,8 @@
             <span class="breadcrump-element">Export CAS Data: </span>
         </div>
     </div>
-    <info-message></info-message>
```

Comment:
This was not needed, under the directive it was hidden for the download page by checking the active url explicitly. Therefore I remove this from here and cleaned up code in info-message.directive.js.